geomesa-spark/geomesa-spark-geotools/src/test/scala/org/locationtech/geomesa/spark/geotools/GeoToolsSpatialRDDProviderTest.scala

```diff
@@ -43,7 +44,7 @@ class GeoToolsSpatialRDDProviderTest extends Specification {
       val ds = DataStoreFinder.getDataStore(dsParams)
       ingestChicago(ds)
 
-      val rdd = GeoMesaSpark(dsParams).rdd(new Configuration(), sc, dsParams, new Query("chicago"))
+      val rdd = GeoMesaSpark(dsParams).rdd[SimpleFeature](new Configuration(), sc, dsParams, new Query("chicago"))
```

Comment:
what about overloading the method? have a `.rdd` with no implicit argument that calls `.rdd[T]` with the simple feature implicit value. (I'm not sure if that would actually work)